glide.lock

```diff
@@ -211,7 +215,7 @@ imports:
   - unicode/bidi
   - unicode/norm
 - name: google.golang.org/genproto
-  version: a7e196e89fd3a3c4d103ca540bd5dac3a736e375
+  version: 32ee49c4dd805befd833990acba36cb75042378c
```

Comment:
this changes from the latest (a7e19) to one from a year ago (32ee49)

Reply:
hmm, that was glide update choosing that. I'll get it to the latest verison

Reply:
sigh, when steven updated this to the latest one, his tests would not pass.

Reply:
This dep is not in any core functionality, only used during the test suite.

Updating it breaks said tests